packages/twenty-server/src/engine/api/graphql/workspace-schema-builder/graphql-types/input/json-filter.input-type.ts

```diff
@@ -0,0 +1,10 @@
+import { GraphQLInputObjectType } from 'graphql';
+
+import { FilterIs } from 'src/engine/api/graphql/workspace-schema-builder/graphql-types/input/filter-is.input-type';
+
+export const JsonFilterType = new GraphQLInputObjectType({
+  name: 'JsonFilter',
+  fields: {
+    is: { type: FilterIs },
```

Comment:
Not sure if this is relevant, this is the type of thing I was referring to when I said it will be easier to test when we have the frontend